planning/behavior_path_planner/src/planner_manager.cpp

```diff
@@ -26,6 +26,42 @@
 
 namespace behavior_path_planner
 {
+bool isInLanelets(const lanelet::ConstLanelets & lanes, const Pose & pose)
+{
+  for (const auto & lane : lanes) {
+    if (lanelet::utils::isInLanelet(pose, lane)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+PathWithLaneId createGoalAroundPath(
```

Comment:
Is it difficult to make these functions common to the one of `behavior_tree_manager.cpp`?

Reply:
Fixed!
https://github.com/autowarefoundation/autoware.universe/pull/3075/commits/5557bec97237162a5998d809966373ff1e85f232